docs/tutorials/introduction.rst

```diff
@@ -1,18 +1,17 @@
 Introduction
 ------------
+.. _Tutorials:
 
-We suggest you download and use the `mu editor <http://codewith.mu/>`_ when
-working through these tutorials. Instructions for downloading and installing
-Mu are on its website. You may need to install a driver, depending on your
-platform (instruction are on the website).
+We suggest you use the `micro:bit Python Editor <https://python.microbit.org>`_
+or `Mu <https://codewith.mu/>`_ when working through these tutorials.
 
-Mu works with Windows, OSX and Linux.
+Connect your micro:bit to your computer via a USB lead.
 
-Once Mu is installed connect your micro:bit to your computer via a USB lead.
+Write your script in the editor window and click the "Download" or "Flash" button to
+transfer it to the micro:bit.
 
-Write your script in the editor window and click the "Flash" button to transfer
-it to the micro:bit. If it doesn't work, make sure your micro:bit appears as
-a USB storage device in your file system explorer.
+If you have any problems with MicroPython or the editor, you can get support from the
+Micro:bit Educational Foundation team via `support.microbit.org <https://support.microbit.org>`_
```

Comment:
I think this sentence needs a full stop at the end:

<img width="726" alt="image" src="https://user-images.githubusercontent.com/29712657/161296874-1b29fe22-b2d3-4172-8f3a-85b24eea0564.png">

Reply:
updated in https://github.com/bbcmicrobit/micropython/pull/732/commits/51b8435df9ce9a08ac82982b012f453efd67f69a